mllib/src/test/scala/org/apache/spark/ml/feature/StopWordsRemoverSuite.scala

```diff
@@ -89,4 +89,22 @@ class StopWordsRemoverSuite
       .setCaseSensitive(true)
     testDefaultReadWrite(t)
   }
+
+  test("StopWordsRemover output column already exists") {
+    val outpuCol = "expected"
+    val remover = new StopWordsRemover()
+      .setInputCol("raw")
+      .setOutputCol(outpuCol)
+      .setCaseSensitive(true)
+    val dataSet = sqlContext.createDataFrame(Seq(
+      (Seq("A"), Seq("A")),
+      (Seq("The", "the"), Seq("The"))
+    )).toDF("raw", outpuCol)
+
+    val thrown = intercept[IllegalArgumentException] {
+      testStopWordsRemover(remover, dataSet)
+    }
+    assert(thrown.getClass === classOf[IllegalArgumentException])
```

Comment:
I tried to take pattern from:
github.com/apache/spark/blob/master/core/src/test/scala/org/apache/spark/FailureSuite.scala#L100-L106

I think this check theoretically could be useful:
https://docs.oracle.com/javase/7/docs/api/java/lang/IllegalArgumentException.html
is non-abstract class and have subclasses,
intercept can catch e.g. PatternSyntaxException, which is not what we should treat as a test success.

But of course, PatternSyntaxException is very unlikely to have message "requirement failed: Column ${outpuCol} already exists."

Am I correct?
Maybe "assert(thrown.getClass === classOf[IllegalArgumentException])" line should stay?